.github/workflows/ci.yml

```diff
@@ -0,0 +1,126 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  merge_group:
+    types:
+      - checks_requested
+
+jobs:
+  lint-and-typecheck:
```

Comment:
The `yarn lint` and `yarn typescript` should both be pretty fast to run?

Do you think we're saving time by running in parallel here?

Reply:
> The `yarn lint` and `yarn typescript` should both be pretty fast to run?
> 
> Do you think we're saving time by running in parallel here?

No I think it's better to run them in sequence